### ISSUE TITLE
The linked schema file is (still) not usable

### ISSUE THREAD

**Issue by StingyJack** [NuGet/docs.microsoft.com-nuget#1371]
I cant import it into VS to use for validating nuspec files like I can do with every other XSD. There are string format placeholders in the namespace attributes at the top that need to be filled out. The namespace seems like something that would change far less frequently than the rest of the file, so I dont get the need for string replacing.

It would be nice if the XSD shipped as part of VS. Its kind of silly that it doesn't. It would have saved me more than a few hours of troubleshooting to find out an element was misspelled or out of place.

I asked for this like two years ago and that request seems to have gone missing.  Please at least fix the XSD so its readily usable. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 06a19d7d-253d-f17f-6421-624c9867473d
* Version Independent ID: cc37eb17-f715-080c-7bfc-9eae868d50ba
* Content: [.nuspec File Reference for NuGet](https://docs.microsoft.com/en-us/nuget/reference/nuspec)
* Content Source: [docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/nuspec.md)
* Product: **nuget**
* GitHub Login: @karann-msft
* Microsoft Alias: **karann**

**Comment by karann-msft** [comment]
that's great feedback. Would it be ok if I moved this issue to the Home repo as this is more of a product feedback?

**Comment by StingyJack** [comment]
No, it would not, for two reasons; there is already an [issue for it there](https://github.com/NuGet/Home/issues/6295) that is in the backlog but stagnant for a long time now. I figured the lack of movement is because the xsd is not really usable for any product purposes as it is and is effectively a documentation artifact, hence this issue. I'm just asking for namespaces to be fixed. If I knew what the correct values were I would have submitted a PR a long time ago.

The other reason is because you have defaced and destroyed my own content, simply because it was critical of a page that attempts to mislead others. Per the rules we all are supposed to follow as part of this community, and as you are just another member as I am and not GH staff, you have/had no right to do so. I rarely make the choice to be offended, but an attempt to quell my right to free speech is incredibly offensive.  Propaganda will always be identifiable by the criticism that follows it.

Instead of all this discussion and the bad feelings generated, the 3 lines in the XSD could just be fixed and we could have moved on years ago. Instead I have to continue to address problems with nuspecs that myself or other teammates have or waste time looking up the correct element names instead of just having a working XSD that I can give to VS that will autocomplete and give me warnings/errors for invalid elements right away. Its a dumb nagging time sink.

**Comment by karann-msft** [comment]
Duplicate of https://github.com/NuGet/Home/issues/6295

**Comment by StingyJack** [comment]
>  the 3 lines in the XSD could just be fixed and we could have moved on...

Gets the sum total response of..

> It is not a documentation artifact. 
> Duplicate of NuGet/Home#6295

@karann-msft - instead of typing that many characters, you could have just typed the namespace that is supposed to be in the `{0}`.  

Or you could recognize the actual ask is to put a component of the nuget system into visual studio in the way it always should have been and has been done for other .net ecosystem xml files. Its not a lot of work for you and would mean we dont have to hack it in or constantly open issues for the nuget team because we have schema problems that a little intellisense would fix.

I really dont understand the constant need for some of the Nuget team to grief me after _already_ creating more unexpected work for me.  Sticking the knife in and then twisting it is not a positive community growth technique.
